test/box/net.box_incorrect_iterator_gh-841.result

```diff
@@ -18,7 +18,7 @@ test_run:cmd("setopt delimiter ';'")
 function x_select(cn, space_id, index_id, iterator, offset, limit, key, opts)
     local ret = cn:_request(remote._method.select, opts, nil, nil, space_id,
                             index_id, iterator, offset, limit, key)
-    return ret
+    return ret[1]
```

Comment:
Can we return one argument if there's no position?

Reply:
What's about the case when` fetch_pos` is true but no position is returned?

Reply:
Done